src/games/cc/backlogmenu.cpp

```diff
@@ -0,0 +1,68 @@
+#include "backlogmenu.h"
+
+#include "../../profile/game.h"
+#include "../../ui/backlogmenu.h"
+#include "../../profile/ui/systemmenu.h"
+#include "../../profile/ui/backlogmenu.h"
+#include "../../profile/games/cc/backlogmenu.h"
+#include "../../profile/games/cclcc/systemmenu.h"
+
+namespace Impacto {
+namespace UI {
+namespace CC {
+
+using namespace Impacto::UI::CC;
+using namespace Impacto::Profile::BacklogMenu;
+using namespace Impacto::Profile::CC::BacklogMenu;
+
+BacklogMenu::BacklogMenu() : UI::BacklogMenu::BacklogMenu() {
+  switch (Impacto::Profile::SystemMenu::Type) {
+    default:
+      MaskSprite = nullptr;
+      break;
+    case UI::SystemMenuType::CCLCC:
+      MaskSprite = &Profile::CCLCC::SystemMenu::MenuMask;
```

Comment:
yeah since we have a separate backlogmenu.lua, it probably would be a good idea to have a separate profile and class for it too, so there's no need to switch on the types

Reply:
You mean add a separate `MenuMask` sprite copy in each component that uses it, and pass it along in the constructor?

Reply:
You're right; should be good now